guardrails/llm_providers.py

```diff
@@ -199,7 +199,11 @@ def _invoke_llm(
         # TODO: Update this to tools
         # Configure function calling if applicable (only for non-streaming)
         fn_kwargs = {}
-        if base_model and not kwargs.get("stream", False):
+        if (
```

Comment:
this preserves the legacy support for the old default support for old openai functions